modules/ocf/manifests/networking.pp

```diff
@@ -39,8 +39,12 @@
   } else {
     # Find the first network interface that starts with 'en', since those are
     # ethernet interfaces. (Won't work for the raspberry pi, since it uses wifi)
-    $ifaces_array = split($::interfaces, ',')
-    $br_iface = grep($ifaces_array, 'en.+')[0]
+    if $::lsbdistid == 'Raspbian' {
+      $br_iface = 'wlan0'
```

Comment:
I would like to find a good way to get this without hardcoding it, but I don't think there's an easy way to do it and we don't want it to use ethernet instead of wifi when configured.

Reply:
What about something like

```puppet
$ifaces_array = split($::interfaces, ',')
$wl_ifaces = grep($ifaces_array, 'wl.+')
$br_iface = length($wl_ifaces) > 0 ? {
  true  => $wl_ifaces[0]
  false => grep($ifaces_array, 'en.+')[0]
}

Reply:
For that matter `br_iface` is a terrible variable name for non-hypervisors

Reply:
Hmmm, yeah that would be better, I'll do that and maybe rename the variable too if I can think of a better name, good idea.